api/v1alpha6/openstackcluster_types.go

```diff
@@ -50,6 +50,9 @@ type OpenStackClusterSpec struct {
 	// through DNS is required.
 	// +listType=set
 	DNSNameservers []string `json:"dnsNameservers,omitempty"`
+	// RouterName is the name of an existing router.
+	// If specified, no router will not be created.
```

Comment:
```suggestion
	// If specified, the router is used and no new router will be created.
```

Reply:
@dhnikolas 
Would be great if you could rephrase this part, as right now it is double negative: `no router will not be created`.